mdakits/lipidorderkit/metadata.yaml

```
@@ -0,0 +1,122 @@
+# TEMPLATE MDAKit file
+# --------------------
+# 
+# Please replace ALL entries with appropriate content for YOUR MDAKit.
+# Below we use the placeholder GH_HOST_ACCOUNT for the GitHub account where
+# the source code repository is held, typically your username or the 
+# organization that you're part off.
+# MYPROJECT is the name of your project (the repository name and here
+# we assume that this is also the PyPi/conda package name) whereas 
+# MYPACKAGE is how you import it in python.
+#
+# See https://mdakits.mdanalysis.org/add.html for more information. 
+#
+#------------------------------------------------------------
+# Required entries
+#------------------------------------------------------------
+## str: name of the project (the respository name)
+project_name: lipidorderkit
+
+## List(str): a link to the authors file (preferred) or a list of authors 
+authors:
+  - https://github.com/ricard1997/lipidorderkit/blob/main/AUTHORS.md
+
+## List(str): a list of maintainers
+## Please note these _must_ be GitHub handles
+## The maintainers will be tagged in issues if their MDAKit is failing.
+maintainers:
+  - Ricardo Ramirez
+
+
+## str: a free form description of the mdakit
+description:
+    This MDAKit allow for the calculation of lipid order parameters for all atom molecualr dynamics simulations.
+
+## List(str): a list of keywords which describe the mdakit
+keywords:
+  - Lipids
+  - Lipids order parameters
+  - SCD
+  - Lipid metrics
+  - Lipid study
+
+## str: the license the mdakit falls under
+## See https://spdx.org/licenses/ for valid license specifiers
+license: GPL-2.0-or-later
+
+## str: the link to the project's code
+## Please note that this is not limited to GitHub! Can be Gitlab, etc..
+project_home: https://github.com/ricard1997/lipidorderkit/
+
+## str: the link to the project's documentation
+documentation_home: https://lipidorderdocs.readthedocs.io/en/latest/
+
+## str: the type of documentation available [UserGuide, API, README]
+documentation_type: UserGuide + API
+
+#------------------------------------------------------------
+# Optional entries
+#------------------------------------------------------------   
+## List(str): a list of commands to use when installing the latest
+## release of the code. Note: only one installation method can currently
+## be defined. We suggest using mamba where possible (e.g.
+##   mamba -c conda-forge install MYPROJECT
+## for a conda package installation).
+## Here we use a simple PyPi installation:
+install:
+  - pip install -e .
+
+## List(str): a list of commands to use when installing the mdakit from its
+## source code.
+src_install:
+  - pip install git+https://github.com/ricard1997/lipidorderkit@main
```

```suggestion
  - git clone https://github.com/ricard1997/lipidorderkit.git
  - cd lipidorderkit && pip install . && cd ..
```

These commands clone the repo and then install the package from source.